specs/template.md

```diff
@@ -10,11 +10,11 @@
 
 ### Use Cases
 
-*Primary use cases for this component.*
+*Primary use cases for this component. These should be based on typical patterns seen in the industry, requests from customers (internal and external), and feedback from tooling.*
```

Comment:
I think we can drop "internal". As far as the project is concerned, all customers are "external" with exception of tooling. Perhaps we can just clarify a bit? Thinking more on this, it seems like we've added the explicit language here to ensure we don't move on without addressing this requirement. While I definitely get the intent of adding it here explicitly, I think it just feels odd to me to call it out as special given tooling gets a vote at the table and can actually hold the spec to ensure it gets in there. 

I'd like to work on a suggestion but I don't have a concise thing *right now*

Reply:
I agree with excluding "internal and external", since both cover the complete intent already, it's just extra words not extra meaning imho.

Reply:
```suggestion
*Primary use cases for this component. These should be based on typical patterns seen in the industry, requests from customers and feedback from tooling.*
```

Reply:
I'm open to the proposed rewording from @janechu, but I do want to push on the fact that just as @janechu is adding feedback to the spec template, she can hold or add *requirements* to any given spec. I get the explicit nod, but it feels odd to me and makes tooling feel disconnected from the overall process. If tooling is required to sign-off on the spec, is this wording necessary?

Reply:
Should we have some wording like "determine if additional requirements and feedback are necessary from tooling"? Because tooling is only impacted with certain components and we don't need to be involved in the process for all of them.

Reply:
@janechu @EisenbergEffect would the language in the first "checkpoint" address this? I see a nod to tooling there and I think if this isn't fully captured there we could update the language for the checkpoint to be clear. The intent of that checkpoint is to ensure that we've captured the use cases. Ensuring the tooling team is invited to that ensures they have a voice and check to validate their scenarios are included. If all scenarios are not, an action item would be to go update the spec to include all relevant needs from the tooling team *can be met*. Thoughts?